misk-feature-testing/src/main/kotlin/misk/feature/testing/TestFeatureFlags.kt

```diff
@@ -0,0 +1,68 @@
+package misk.feature.testing
+
+import com.google.common.util.concurrent.AbstractIdleService
+import misk.feature.Attrs
```

Comment:
Please rename this

Reply:
Convention in Java land is to not abbreviate things. Motivation is that abbreviations are hard to guess and hard to grep

Reply:
Standard abbreviations are fine no? I don't think anyone would be confused what Attr means. We use HTTP and RPC all over the place. RPC is more alien than Attr imo

Reply:
I'm recalling Effective Java Item 68. This is how we’ve named most of the things in Misk and I think it’s also good for us to be consistent!

Reply:
(Abbreviations and Acronyms are different things, and the rules of acronyms is that they’re written like words: XmlHttpRequest instead of XMLHTTPRequest.)